python/setup.cfg

```diff
@@ -48,3 +49,11 @@ skip=
     build
     dist
     __init__.py
+
+
+[options]
+packages = find:
+install_requires =
+    numpy
+    numba>=0.49
+python_requires = >=3.6,<3.8
```

Comment:
I think this is incorrect, we support 3.8 and don't support 3.6 anymore. So maybe `>=3.7,<3.9`?

Reply:
Corrected it across code-base.